src/Tribe/Views/V2/Views/Traits/With_Fast_Forward_Link.php

```diff
@@ -32,15 +32,15 @@ trait With_Fast_Forward_Link {
 	 * @return string                  The html link and message.
 	 */
 	public function get_fast_forward_link( $canonical = false, array $passthru_vars = [] ) {
-		if ( $this->use_ff_link( $canonical, $passthru_vars ) ) {
+		if ( ! $this->use_ff_link( $canonical, $passthru_vars ) ) {
 			return '';
 		}
 
 		$date      = $this->context->get( 'event_date', $this->context->get( 'today' ) );
 		$cache_key = __METHOD__ . '_' . md5( wp_json_encode( array_merge( [ $date, $canonical ], $passthru_vars ) ) );
 
 		if ( isset( $this->cached_urls[ $cache_key ] ) ) {
-			return $this->cached_urls[ $cache_key ];
+			//return $this->cached_urls[ $cache_key ];
```

Comment:
* :warning: This comment is 64% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// return $this->cached_urls[ $cache_key ];" but found "//return $this->cached_urls[ $cache_key ];"
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 629d0b0f67363781d230f157ce3a52de</sub>